tests/sentry/workflow_engine/migrations/test_0050_remove_orphaned_rule_workflows.py

```diff
@@ -0,0 +1,103 @@
+from sentry.testutils.cases import TestMigrations
+from sentry.workflow_engine.models import (
+    Action,
+    AlertRuleWorkflow,
+    DataCondition,
+    DataConditionGroup,
+    DataConditionGroupAction,
+    Workflow,
+    WorkflowDataConditionGroup,
+)
+from sentry.workflow_engine.models.data_condition import Condition
+
+
+class TestMigrateIssueAlerts(TestMigrations):
```

Comment:
Once you've made the change I suggested and validated against this test, I'd recommend just commenting out this test. Until we finish squashing the migrations there's a chance that this will end up timing out and breaking ci.